style-test.py

```diff
@@ -235,8 +303,8 @@ def get_paths(paths):
 
 def get_changed_files():
     """Return currently modified rst files."""
-    file_path = os.path.realpath(__file__)
-    repo_path = file_path[0:file_path.rfind('/')]
+    global dir_path
+    repo_path = dir_path
     repo = git.Repo(repo_path)
     changedFiles = [item.a_path for item in repo.index.diff(None)]
```

Comment:
Not a big deal, but I should point out that you switched to a different style here:
`changedFiles` instead of `changed_files`

(Incidentally, the `underscore_style` is considered standard in Python for variables and function names. The only thing you typically use CamelCase for is class names.